geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -2795,21 +2708,18 @@ public LogWriterI18n getSecurityLoggerI18n() {
     return system.getSecurityInternalLogWriter();
   }
 
+  @Deprecated
```

Comment:
Should these be deprecated on `InternalCache` instead?

Reply:
These are all deprecated in the interface with both javadoc and annotation. I copied the Deprecated annotation down to the impl to help the reader realize they're looking at a deprecated method.